Orderbook tutorial (#254)
sora-xor/sora-docs: src/orderbook-trading.md

---
title: "A Comprehensive Guide to the Polkaswap Decentralized Order Book | SORA Docs"
head:
  - - meta
    - name: description
      content: "Quickly master the Polkaswap Order Book: connect your wallet, navigate the DEX, and trade effectively. Learn the order book interface for efficient decentralized trading."
  - - meta
    - name: keywords
      content: "DEX, order book, decentralized, spot trading, Polkadot, limit order"
---

# Polkaswap Advanced Trading: Order Book

![](.gitbook/assets/polkaswap-orderbook-tutorial-hero.png)

## Theory

An order book is a fundamental component of a decentralized exchange (DEX) that facilitates trading digital assets between buyers and sellers in a transparent and efficient manner. It serves as a virtual meta ledger that records all the buy and sell orders placed by participants in the market.

This tutorial will walk you through the essential steps for trading on Polkaswap, focusing on connecting your wallet, understanding the interface, and executing trades. Let's get started.

> :::info

Reviewer: ```suggestion
::: info
```

> For advanced Orderbook functionality as well as how to integrate the SORA Orderbook in your dApp, refer to the [Orderbooks](orderbooks.md) section.

Reviewer: ```suggestion
For advanced Orderbook functionality as well as for detail on how to integrate the SORA Orderbook in your dApp, refer to the [Orderbooks](orderbooks.md) section.
```

:::

## Practice

1. **Connect Your Wallet:** To begin trading, connect your wallet to Polkaswap. This is a crucial step to access your funds and initiate trades.

2. **Fund Your Wallet:** Ensure you have XOR or another cryptocurrency. You'll need these assets to start trading on Polkaswap.

3. **Navigate to the Trade Tab:** Here's where the action happens. Click on the Trade tab to access the trading interface.

![](.gitbook/assets/polkaswap-orderbook-interface.png)

### Explore the Interface:

You'll see several key components in the Orderbook interface:

- **The Order Book Widget:** Shows all the aggregated orders for a selected pair.

> - **Charts:** Visual representations of trading activity.
> - **Buy and Sell Widgets:** Where you'll place your orders.

Reviewer: ```suggestion
- **Charts:** Shows visual representations of trading activity.
- **Buy and Sell Widgets:** Shows where you'll place your orders.
```

- **History Widget:** Displays your past trading activities.
- **Market Trades Widget:** Shows all trades for the selected pair.
- **Understanding Charts:** The chart for the selected pair reflects data related to Order Book trading. It does not aggregate liquidity from swaps.

### Placing Orders:

Choose the token pair you wish to trade. There are two types of orders:

- **Market Order:** Executes at the latest pair price, similar to a swap.

![](.gitbook/assets/polkaswap-orderbook-market.png)

- **Limit Order:** Allows you to set any desired price. Orders default to one month and will be automatically canceled after this period.

![](.gitbook/assets/polkaswap-orderbook-limit.png)

> 4. **Order Cancellation:** You can cancel your order at any time

Reviewer: why this list starts at 4?

Author: its a continuation of the ongoing list at the beginning of the document

   without any fees.

5. **Fees on Order Placement:** You will incur a fee when placing an

>    order. The fee amount is dependent on the price, similar to current

Reviewer: ```suggestion
   order. The fee amount depends on the price, similar to current
```
